source/MaterialXView/Viewer.cpp

```diff
@@ -830,6 +845,42 @@ void Viewer::createAdvancedSettings(Widget* parent)
             _envSamples = MIN_ENV_SAMPLES * (int) std::pow(4, index);
         });
     }
+
+    ng::Label* textureLabel = new ng::Label(advancedPopup, "Texture Baking Options");
+    textureLabel->setFontSize(20);
+    textureLabel->setFont("sans-bold");
+
+    ng::CheckBox* bakeTexturesBox = new ng::CheckBox(advancedPopup, "Bake Textures");
+    bakeTexturesBox->setChecked(_bakeTextures);
+    bakeTexturesBox->setCallback([this](bool enable)
+        {
```

Comment:
For consistency, the braces for these lambda closures should be aligned with the function call, as in:

```
bakeTexturesBox->setCallback([this](bool enable)
{
    _bakeTextures = enable;
});
```